README.md

```diff
@@ -7,50 +7,24 @@
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- React
+- CSS
+- Gitflow
 
 ## Live Demo (if available)
 
 [Live Demo Link](https://livedemo.com)
 
 
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
```

Comment:
- [ ] As stated by the previous reviewer, kindly add specific instructions on how to set up and run the project locally. i.e. 
  - clone the repo
  - cd into the directory
  - run `npm install` to install packages
  - run `npm start` to run the app